Misc/NEWS.d/next/Security/2024-12-05-21-35-19.gh-issue-127655.xpPoOf.rst

```diff
@@ -0,0 +1 @@
+Fixed the :mod:`asyncio` selector transport not pausing writes for the protocol when the buffer reaches the high water mark when using ``writelines``.
```

Comment:
If it's possible (but not sure), would you be willing to add a link to `writelines` so that users know which "writelines" method you're talking about? [is it https://docs.python.org/3/library/asyncio-stream.html#asyncio.StreamWriter.writelines?]